### DISCUSSION TITLE
`ColorType::Rgb32`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [image-rs/image#2054] [Q&A] [answered]
Hello guys!
May I ask if there is something like `ColorType::Rgb32` (basically the same as `ColorType::RgbF32` but instead of a float it's an integer).

Would it be worth it to add this to the enum in a PR?

**Comment by fintelia** [comment]
There isn't currently, because we try to limit `ColorType` to only the most common. What use case do you have for Rgb32?

**Reply by TornaxO7** [reply]
I'm trying to write a little screenshot tool with my friend and if I'm seeing it correctly the rgb-values for 24bits to 32-bits are full integers as that's how [screenshots-rs](https://github.com/nashaofu/screenshots-rs/blob/master/src/linux/xorg.rs#L53-L68) does it.

**Reply by fintelia** [reply, marked as answer]
I think you might be confusing bits per sample with bits per pixel. Rgb8 is 8 bits per sample, but 24-bits per pixel. RgbF32 is 32 bits per sample / 96-bits per pixel.

**Reply by TornaxO7** [reply]
oooooooooooooooooooooh, yes, I think you're right!

**Reply by TornaxO7** [reply]
Thank you! That was indeed the issue c(^-^)c
